### ISSUE TITLE
ERROR tests/test_verify_base_url.py

### ISSUE THREAD

**Issue by carlosal1015** [pytest-dev/pytest-base-url#27]
Hi, over Arch Linux we have the following error with recent [`python 3.10.2-1`](https://archlinux.org/packages/core/x86_64/python). Full log is [here](https://gist.github.com/carlosal1015/49f85df45898789e90d5faf602341986).
```python
============================= test session starts ==============================
platform linux -- Python 3.10.2, pytest-6.2.5, py-1.11.0, pluggy-0.13.1
rootdir: /tmp/makepkg/python-pytest-base-url/src/pytest-base-url, configfile: tox.ini, testpaths: tests
plugins: base-url-1.4.2, localserver-0.5.1
collected 10 items

tests/test_base_url.py .....                                             [ 50%]
tests/test_verify_base_url.py ..E.E..E                                   [100%]

==================================== ERRORS ====================================
_______________ ERROR at teardown of test_enable_verify_via_cli ________________

cls = <class '_pytest.runner.CallInfo'>
func = <function call_runtest_hook.<locals>.<lambda> at 0x7f083c56b880>
when = 'teardown'
reraise = (<class '_pytest.outcomes.Exit'>, <class 'KeyboardInterrupt'>)

    @classmethod
    def from_call(
        cls,
        func: "Callable[[], TResult]",
        when: "Literal['collect', 'setup', 'call', 'teardown']",
        reraise: Optional[
            Union[Type[BaseException], Tuple[Type[BaseException], ...]]
        ] = None,
    ) -> "CallInfo[TResult]":
        excinfo = None
        start = timing.time()
        precise_start = timing.perf_counter()
        try:
>           result: Optional[TResult] = func()

/usr/lib/python3.10/site-packages/_pytest/runner.py:311: 
```

**Comment by BeyondEvil** [comment]
Thanks for reporting and sorry for the delay in response.

Looking at the full log real quick I get the sense the issue might be with requests.

But I'll see if I can repro it.

**Comment by carlosal1015** [comment]
Thank you, in the morning I will share a gitpod environment or dockerfile for reproduce.

**Comment by BeyondEvil** [comment]
I just published `v2.0.0` and it should solve the issue.

**Comment by carlosal1015** [comment]
Testing the new version for upload to AUR, news very soon.
![image](https://user-images.githubusercontent.com/21283014/160291049-ce733c6f-c64b-4ab5-908d-f91fc82d67af.png)
